src/components/BlogPost/BlogPost.js

```diff
@@ -46,24 +100,51 @@ function BlogPost(props) {
           <Grid>{blogSideDate}</Grid>
           {blogSideTags}
         </Grid>
-        <Grid item xs={9}>
+        <Grid item lg={9} xs={12}>
           <Link
+            className={width === 'xs' ? classes.linkSmall : classes.link}
             to={{
               pathname: `/blogs/${id}`,
             }}
           >
-            <h1>{name}</h1>
+            <span>{name}</span>
           </Link>
+          {/* <div
+            className={props.width === 'xs' ? classes.red : classes.blue}
+          >{`Current width: ${props.width}`}</div> */}
           {image && <img className={classes.image} src={image} alt={alt} />}
-          <p className={classes.description}>{description}</p>
+          <p
+            className={width === 'xs' ? classes.descSmall : classes.description}
+          >
+            {description}
+          </p>
           <div className={classes.mediaContainer}>{link}</div>
         </Grid>
       </Grid>
-      <Divider />
+      <Divider
+        className={width === 'xs' ? classes.dividerSmall : classes.divider}
+      />
     </Container>
   );
 }
 
-export default withStyles(styles)(BlogPost);
+BlogPost.defaultProps = {
+  image: '',
+  alt: '',
+};
 
-//final stop
\ No newline at end of file
+// export default withStyles(styles)(BlogPost);
+BlogPost.propTypes = {
+  classes: PropTypes.object.isRequired,
+  blogSideDate: PropTypes.string.isRequired,
+  blogSideTags: PropTypes.string.isRequired,
+  name: PropTypes.string.isRequired,
+  image: PropTypes.string,
+  alt: PropTypes.string,
+  description: PropTypes.object.isRequired,
```

Comment:
Here you are saying description is an object that is required. In blogs.js line 416, you're passing description as a string. Generally, when you map through an array to produce a list of components, you want to pass your props consistently or you might run into some errors that can break your site (If you're curious, look up why type checking is important). If you need to have it be another type, you can do
```
size: PropTypes.oneOfType([
  PropTypes.string,
  PropTypes.number
]),
```

Here you can do that, or fix the place where you're passing a string ```line 416 blogs.js``` instead of an object.

Reply:
changed string in line 416 to an object